ssh-server/main.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bytes"
+	"encoding/binary"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+
+	"github.com/kr/pty"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	serverPort = flag.Uint("port", uint(22), "the port to listen on")
```

Comment:
Best to name flags the same as their variables unless you have a good reason to otherwise (eg, port instead of serverPort)

Reply:
done